ui/src/app/console/Console.tsx

```diff
@@ -223,6 +303,17 @@ export default function Console() {
                         </div>
                       </div>
                       <div className="flex justify-end">
+                        <Button
+                          primary
```

Comment:
```suggestion
```
I dont think we want two primary buttons, the 'Evaluate' should still be the primary IMO and this secondary